scipy/linalg/special_matrices.py

```diff
@@ -1194,3 +1194,90 @@ def fiedler_companion(a):
     c[[0, 1], 0] = [-a[1], 1]
 
     return c
+
+def convolution_matrix(a, n, mode='full'):
+    """
+    Construct a convolution matrix.
+
+    Constructs the dense matrix representing convolution[1].
+
+    `A = convolution_matrix(a, n[, mode])`
```

Comment:
All codelike backtick guards need to be doubled as single backtick means a Sphinx link directive to link to other functions for example to have a link to another function \`scipy.linalg.toeplitz\`

Reply:
OK, I think I've followed the single-vs-double backtick convention